src/controllers/index.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const overview = require('./overview');
+const setting = require('./setting');
+
+const router = express.Router();
+
+// Home Page //
```

Comment:
delete comments

Reply:
I think this is important to prevent the conflict and to order the file
What do you think @HemaSAli